config/envs/test.py

```diff
@@ -15,3 +15,12 @@ class TestConfig(DefaultConfig):
     RSA256_PUBLIC_KEY = base64.b64decode(
         environ.get("RSA256_PUBLIC_KEY_BASE64")
     ).decode()
+
+    # Redis Feature Toggle Configuration
+    REDIS_INSTANCE_NAME = "funding-service-magic-links-test"
+    REDIS_INSTANCE_URI = (
+        DefaultConfig.VCAP_SERVICES.get_service_credentials_value(
+            "redis", REDIS_INSTANCE_NAME, "uri"
+        )
+    )
+    TOGGLES_URL = REDIS_INSTANCE_URI + "/0"
```

Comment:
`TOGGLES_URL` is the same everywhere so can we move to default?

Reply:
done!